### PR TITLE
Add Converters addresses for Sepolia and Ethereum

### PR DESCRIPTION
**Pull request by Debugger022**
This PR adds converters addresses on `Ethereum` and `Sepolia`.

### CODE REVIEW

**Review by GitGuru7 — APPROVED** [review]
Addresses LGTM

**Review by chechu — APPROVED** [review]

File: technical-reference/reference-technical-articles/token-converters.md [hunk 1]
```diff
@@ -12,16 +12,30 @@ The [Token Converter Bot](https://github.com/VenusProtocol/keeper-bots/blob/deve
...
-- [VTreasury](https://bscscan.com/address/0xf322942f644a996a617bd29c16bd7d231d9f35e9)
+The **Venus Protocol** generates income in various underlying tokens from interest and liquidation fees, that are then sent to the ProtocolShareReserve contract. The `ProtocolShareReserve` contract disburses these earnings to number of destinations.
+
+## Income Distribution
+
+- ## BNB Chain
+
+| Converter               | Accepts           | Interest Reserves (%)| Liquidation Income (%)  |
+|-------------------------|-------------------|----------------------|-------------------------|
+| RiskFundConverter       | 	USDT          | 	40%	             |                     50% |
```
Comment (chechu) [inline technical-reference/reference-technical-articles/token-converters.md]: After the VIP-332, the RiskFundConverter doesn't receive any funds from the PSR

```suggestion
```

File: technical-reference/reference-technical-articles/token-converters.md [hunk 2]
```diff
@@ -12,16 +12,30 @@ The [Token Converter Bot](https://github.com/VenusProtocol/keeper-bots/blob/deve
...
+| XVSVaultConverter       | 	XVS	          |     10%	             |                     10% |
```
Comment (chechu) [inline technical-reference/reference-technical-articles/token-converters.md]: This changed in the VIP-332

```suggestion
| XVSVaultConverter       | 	XVS	          |     20%	             |                     20% |
```

File: technical-reference/reference-technical-articles/token-converters.md [hunk 3]
```diff
@@ -12,16 +12,30 @@ The [Token Converter Bot](https://github.com/VenusProtocol/keeper-bots/blob/deve
...
+| USDTPrimeConverter      | 	USDT          | 	4%	             |                      0% |
+| USDCPrimeConverter      | 	USDC          | 	4%	             |                      0% |
+| BTCBPrimeConverter      | 	BTCB          | 	0.5%             |                      0% |
+| ETHPrimeConverter       | 	ETH	          |     1.5%             |                      0% |
```
Comment (chechu) [inline technical-reference/reference-technical-articles/token-converters.md]: This changed in the VIP-337:

```suggestion
| USDTPrimeConverter      | 	USDT          | 	9%	             |                      0% |
| USDCPrimeConverter      | 	USDC          | 	7%	             |                      0% |
| BTCBPrimeConverter      | 	BTCB          | 	1%             |                      0% |
| ETHPrimeConverter       | 	ETH	          |     3%             |                      0% |
```

Comment (Debugger022) [reply technical-reference/reference-technical-articles/token-converters.md]: Done https://github.com/VenusProtocol/venus-protocol-documentation/pull/213/commits/f0316990cf4f5a8380f3d470f9ff442f94235253

File: technical-reference/reference-technical-articles/token-converters.md (hunk 2 above)
Comment (Debugger022) [reply technical-reference/reference-technical-articles/token-converters.md]: Done https://github.com/VenusProtocol/venus-protocol-documentation/pull/213/commits/f0316990cf4f5a8380f3d470f9ff442f94235253

File: technical-reference/reference-technical-articles/token-converters.md (hunk 1 above)
Comment (Debugger022) [reply technical-reference/reference-technical-articles/token-converters.md]: Done https://github.com/VenusProtocol/venus-protocol-documentation/pull/213/commits/f0316990cf4f5a8380f3d470f9ff442f94235253

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**